recaptcha_enterprise/snippets/src/main/java/recaptcha/account_defender/AccountDefenderAssessment.java

```diff
@@ -84,19 +79,26 @@ public static void accountDefenderAssessment(
       String recaptchaSiteKey,
       String token,
       String recaptchaAction,
-      ByteString hashedAccountId)
+      String accountId,
+      String phoneNumber,
+      String emailAddress)
       throws IOException {
     try (RecaptchaEnterpriseServiceClient client = RecaptchaEnterpriseServiceClient.create()) {
 
       // Set the properties of the event to be tracked.
-      Event event =
+      Event.Builder eventBuilder =
```

Comment:
Can we combine two separate property settings into one instance like:

Event event = Event.newBuilder()
                               .setSiteKey()
                               .setToken()
                               .setAcountId()
.....

Reply:
we can't because of the